{{cookiecutter.project_slug}}/README.md

```diff
@@ -76,40 +76,25 @@ can access it at [http://localhost:3000/](http://localhost:3000/).
 ## SealedSecrets for passwords and sensitive values
 
 SealedSecrets can be used to encrypt passwords for the values to be safely checked in.
-To create a new secret encrypt the base64 encoded secrets using [kubeseal](https://github.com/bitnami-labs/sealed-secrets#kubeseal), for example:  
-**unsealed_secrets.yaml**
-```
-apiVersion: v1
-data:
-  SECRET_NAME: BASE64_ENCODED_SECRET
-kind: Secret
-metadata:
-  name: secrets-config
-  namespace: {{cookiecutter.project_dash}}
-type: Opaque
-```
-
-Configure kubernetes to use your project config and context
+To create a new secret encrypt the base64 encoded secrets using [kubeseal](https://github.com/bitnami-labs/sealed-secrets#kubeseal).  
+
+Configure kubernetes to your current project config and context, making sure you are in the correct prod/sandbox environment
 
     $ export KUBECONFIG=~/.kube/config:~/.kube/{{cookiecutter.project_slug}}.ec2.config
     $ kubectl config use-context {{cookiecutter.project_slug}}-ec2-cluster 
 
-Output the encrypted secrets to your kubernetes manifest  
+You can store the secrets in 1Password and read the sensitive values to set it as enviroment variables:
+(The 1Password path in .envrc must match the path in the vault)
+
+    $ make read-op-secrets
```

Comment:
It is unclear what must be done.
```suggestion
To add a secret to your project, edit k8s/secrets.yaml.template and add a new secret e.g.:

    data:
       SECRET_NAME: ${SECRET_NAME}

Note that we use the ${VARIABLE} pattern for the secret value since we use `envsubst` to substitute the secret value when creating the secrets manifest.

Run `make prod-secrets` to generate the secrets manifets.

```